cmdlist.go

```diff
@@ -0,0 +1,56 @@
+package main
+
+import (
+	"os"
+
+	"github.com/rogpeppe/modcop/depfile"
+)
+
+var listCommand = &Command{
+	Short:     "list dependencies",
+	UsageLine: "list [packages]",
+	Long: `
+List shows the set of current dependencies of the listed packages.
```

Comment:
I would clarify this a bit further, like the first line of `go help list`:

```
List lists the named packages, one per line.
```

Reply:
The clarification being that you are getting line-based output

Reply:
done